.changeset/search-cross-the-goal.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/search-common': patch
+'@backstage/plugin-search-backend-node': patch
+'@backstage/plugin-search': patch
+---
+
+The `<Search...Next /> set of components exported by the Search Plugin are now updated to use the Search Backend API. These will be made available as the default non-"next" versions in a follow-up release.
+
+The interfaces for decorators and collators in the Search Backend have also seen minor, breaking revisions ahead of a general release. If you happen to be building on top of these interfaces, check and update your implementations accordingly. The APIs will be considered more stable in a follow-up release.
```

Comment:
Yeah, that will be the plan in #5716.  Patch now (because everything is kinda unstable, the default, front-end only search experience is still maintained).  But minor bump and changeset when we make this experience the default experience.